### ISSUE TITLE
v0.5.0 widget does not appear

### ISSUE THREAD

**Issue by rjsmith** [meteor-velocity/html-reporter#69]
Upgrading `velocity:html-reporter` package to `@0.5.0` causes the Velocity widget to disappear from the application UI.

This can be replicated in an existing app and on a brand - new meteor project:

```
meteor create test-foo
cd test-foo
meteor add velocity:html-reporter
meteor add sanjo:jasmine
meteor
```
Navigate to app at localhost:3000
The Velocity target is missing

.meteor/versions:

```
amplify@1.0.0
autopublish@1.0.3
autoupdate@1.2.1
base64@1.0.3
binary-heap@1.0.3
blaze@2.1.2
blaze-tools@1.0.3
boilerplate-generator@1.0.3
callback-hook@1.0.3
check@1.0.5
coffeescript@1.0.6
ddp@1.1.0
deps@1.0.7
ejson@1.0.6
fastclick@1.0.3
geojson-utils@1.0.3
html-tools@1.0.4
htmljs@1.0.4
http@1.1.0
id-map@1.0.3
insecure@1.0.3
jquery@1.11.3_2
json@1.0.3
launch-screen@1.0.2
less@1.0.14
livedata@1.0.13
logging@1.0.7
meteor@1.1.6
meteor-platform@1.2.2
minifiers@1.1.5
minimongo@1.0.8
mobile-status-bar@1.0.3
mongo@1.1.0
observe-sequence@1.0.6
ordered-dict@1.0.3
package-version-parser@3.0.3
practicalmeteor:chai@1.9.2_3
practicalmeteor:loglevel@1.1.0_3
random@1.0.3
reactive-dict@1.1.0
reactive-var@1.0.5
reload@1.1.3
retry@1.0.3
routepolicy@1.0.5
sanjo:jasmine@0.12.7
sanjo:karma@1.4.3
sanjo:long-running-child-process@1.0.3
sanjo:meteor-version@1.0.0
session@1.1.0
spacebars@1.0.6
spacebars-compiler@1.0.6
templating@1.1.1
tracker@1.0.7
ui@1.0.6
underscore@1.0.3
url@1.0.4
velocity:core@0.4.5
velocity:html-reporter@0.5.0
velocity:meteor-stubs@1.0.3
velocity:node-soft-mirror@0.3.1
velocity:shim@0.1.0
velocity:test-proxy@0.0.4
webapp@1.2.0
webapp-hashing@1.0.3
```

**Comment by carlosbaraza** [comment]
:+1:

**Comment by ryanswapp** [comment]
Is this fixed yet? I decided to teach myself how to test a Meteor app today and have been pulling my hair out for the last hour trying to figure out why the html reporter widget isn't showing up in the top right corner of my app :( 

I am running Meteor 1.1.0.2 and velocity:html-reporter@0.5.0

**Comment by samhatoum** [comment]
doing it right now!

**Comment by ryanswapp** [comment]
Awesome, thanks! Also, I just downgraded to 1.1.0.1  and then 1.1 and it still does not work... Do you think that I might have done something to mess it up? All I have done is install sanjo:jasmine and the velocity reporter and add one test. The test output shows up in the console if I use the console reporter.

**Comment by samhatoum** [comment]
no, I did! You can downgrade the reporter to 4.3 that should do it

**Comment by ryanswapp** [comment]
After just trying a few different commands I'm not sure I know how to downgrade a specific package... How would I downgrade the reporter to 4.3?

**Comment by samhatoum** [comment]
in your .meteor/packages file, set `velocity:html-reporter@0.4.3`

**Comment by ryanswapp** [comment]
I tried that earlier with no luck and just tried again... No dice. Here is the link to my apps repo if you have time to take a look https://github.com/ryanswapp/vurvy

**Comment by Sanjo** [comment]
Do you see an html-reporter error in the web console (e.g Chrome Dev Tools)?

**Comment by rjsmith** [comment]
No error, the DOM element is simply not added in 0.5.0. Could be problem in html-reporter template helper logic that determines if widget should be shown?

**Comment by samhatoum** [comment]
The issue is that this call is returning 404

``` 
Meteor.call('velocity/isEnabled', function(error, result){
    Session.set('velocity.isEnabled', result);
  });
```

I think the current version of velocity (4.5) does not have that method, but I got fooled into thinking it existed as it was on the develop branch.

A quick fix would be to set this session var in your app somewhere:

`Session.set('velocity.isEnabled', true);`

**Comment by Sanjo** [comment]
So we just increase the minimum required version of velocity:core in html-reporter?

**Comment by samhatoum** [comment]
just released 0.5.3, this should work with old velocity versions.

**Comment by Sanjo** [comment]
Tested with the reproduce from @rjsmith. Works.
